core/src/main/java/com/infinum/jsonapix/core/common/JsonApiXMissingArgumentException.kt

```diff
@@ -0,0 +1,8 @@
+package com.infinum.jsonapix.core.common
+
+import java.lang.IllegalArgumentException
+
+// todo NIKOLA add attribute logging
+class JsonApiXMissingArgumentException : IllegalArgumentException() {
```

Comment:
As we discussed, I think it would be completely fine, to keep all the custom exception in one file. Also maybe you could add that attributte logging here.

Reply:
Yeah, forgot about this as well 👍

Reply:
The reason why `exceptions` is a problem is due to 2 conflicting detekt rules. FileName for formatting and MatchingDeclarationName for naming. I would shut this down but I need a custom detekt config to do so. I would leave it like it is for now with adding an attribute but keep the naming for now until I add a custom detekt config. How does this sound?

Reply:
Sounds good 👍